common/djangoapps/student/tests/test_recent_enrollments.py

```diff
@@ -126,6 +127,33 @@ def test_dashboard_rendering(self):
         response = self.client.get(reverse("dashboard"))
         self.assertContains(response, "Thank you for enrolling in")
 
+    def test_dashboard_escaped_rendering(self):
+        """
+        Tests that the dashboard renders the escaped recent enrollment messages appropriately.
+        """
+        self._configure_message_timeout(600)
+        self.client.login(username=self.student.username, password=self.PASSWORD)
+
+        # New Course
+        course_location = locator.CourseLocator('TestOrg', 'TestCourse', 'TestRun')
+        xss_content = "<script>alert('Test')</script>"
+        course = CourseFactory.create(
+            org=course_location.org,
+            number=course_location.course,
+            run=course_location.run,
+            display_name=xss_content
+        )
+        enrollment = CourseEnrollment.enroll(self.student, course.id)
+        enrollment.created = datetime.datetime.now(UTC)
```

Comment:
Why you need to change the `created`? I suppose calling `CourseEnrollment.enroll(self.student, course.id)` adds the latest date itself and there is no point of updating? 
Can you confirm?